Systematics/python/SystematicDumperDefaultVariables.py

```diff
@@ -9,7 +9,7 @@
 
 minimalNonSignalVariables = ["CMS_hgg_mass[160,100,180]:=diPhoton().mass"]#,"centralObjectWeight[1,-999999.,999999.] := centralWeight"]
 
-minimalVariablesHTXS = minimalVariables+["stage0cat[72,9.5,81.5] := tagTruth().HTXSstage0cat"]
+minimalVariablesHTXS = minimalVariables+["stage0bin[72,9.5,81.5] := tagTruth().HTXSstage0bin"]
```

Comment:
these are the defaults used if doHTXS is switched on but doStageOne is not. Wanted to keep them because e.g. for the ttH analysis I believe this is the configuration they used. Can remove if preferred of course.

Reply:
ok